src/Utilities.jl

```diff
@@ -188,4 +188,30 @@ function time_to_seconds(s::String)
 end
 
 
+"""
+    integral(field)
+
+Return the integral for `field` along its spatial dimensions.
+"""
+function integral(field::CC.Fields.Field)
```

Comment:
We could add a test for this function that `sum(2d field) == integral(level field)` since that's the problem we're trying to solve with it